ocean_lib/common/aquarius/aquarius.py

```diff
@@ -85,6 +74,17 @@ def get_asset_ddo(self, did):
 
         return DDO(dictionary=parsed_response)
 
+    def ddo_exists(self, did):
```

Comment:
Since this function returns a `bool`, can we provide the return type in the function signature such as `def ddo_exists(self, did) -> bool`?

Reply:
yes, when we fix the conflicts with the typing branch :))